### DISCUSSION TITLE
Need help confirming exploit

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#839] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I recently found something where you can unblock crosh if its blocked by goguardian, asked my friend to see if its just me or smth, he said it didnt work. So here i am
Step 1: ctrl+alt+t
step 2: do it again, but now spam reload after crosh tab opens
done

**Comment by 3kh0** [comment, marked as answer]
Not able to reproduce

**Reply by ghost** [reply]
k, thanks
